### PR TITLE
DocumentWorkflow interrogators

### PR DESCRIPTION
**Pull request by whoojemaflip**
helper methods to interrogate workflow_state

### CODE REVIEW

**Comment by cesidio** [comment]
👍

File: spec/services/document_workflow_spec.rb [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+  let(:initial_status) { 'not_started' }
+
+  describe "state_interrogator" do
+    shared_examples "state_interrogator" do |state|
+      let(:result) { subject.public_send("is_#{state}?") }
+
+      context "model is in #{state} state" do
+        let(:initial_status) { state }
+
+        it "is true" do
```
Comment (cesidio) [inline spec/services/document_workflow_spec.rb]: what about

its(:result) { is_expected.to be true }

Comment (dan-palmer) [inline spec/services/document_workflow_spec.rb]: A similar alternative without relying on the its gem is RSpecs specify

**Comment by dan-palmer** [comment]
:+1:

**Pull request merged**